src/movegen.cpp

```diff
@@ -398,7 +398,11 @@ namespace {
             while (b)
                 *moveList++ = make_move(ksq, pop_lsb(&b));
         }
+#ifdef SUICIDE
         if (pos.is_suicide() || pos.can_capture())
+#else
+        if (pos.can_capture())
```

Comment:
This is within the `#ifdef ANTI` scope, so there should not be a problem. It would probably anyway be good to test different combinations of enabled variants to make sure that I have not introduced a bug somewhere.